dot/network/service_test.go

```diff
@@ -54,9 +56,36 @@ func createServiceHelper(t *testing.T, num int) []*Service {
 	return srvcs
 }
 
+func createTestBlockResponseMessage(t *testing.T) *BlockResponseMessage {
+	t.Helper()
+
+	msg := &BlockResponseMessage{
+		BlockData: []*types.BlockData{},
```

Comment:
nit not that it matters, but you could pre-allocate with `make([]*types.BlockData, blockRequestSize)` and assign directly using the index instead of using `append`. But it doesn't matter though :wink: